packages/storage-plus/src/snapshot.rs

```diff
@@ -284,10 +285,15 @@ mod tests {
 
     #[test]
     fn namespace_macro() {
-        let names = snapshot_names!("demo");
-        assert_eq!(names.pk, b"demo");
-        assert_eq!(names.checkpoints, b"demo__checkpoints");
-        assert_eq!(names.changelog, b"demo__changelog");
```

Comment:
This works if you put the clippy allow before the macro def (see above).

Reply:
Better said, this works for me, and without the allow macro. I'm using rust 1.48.0 stable, though.

Reply:
You migh want to take a look at this, as I've now tried with 1.47.0, and it works. After removing the clippy allow, which throws an `error[E0658]: attributes on expressions are experimental`.

Reply:
I tried with 1.47.0.

The clippy problem never was with this line of code so much, but rather with compiling `cw4-group`. I only get the error there (as it is in non-test code)

Reply:
Ah, OK. Will take a look tomorrow.